themes/Suite7/css/style.css

```diff
@@ -5821,6 +5821,7 @@ table tr#delegates_search {
 /* fix schedule bar popup on calendar */
 .ui-front, .ui-dialog {
     z-index: 1100 !important;
+    left: 1531px !important;
```

Comment:
Left is very specific but surely this would change depending on the side of the window and number of columns on the list view?